### ISSUE TITLE
using kubectl logs -f order,Error from server: Get...

### ISSUE THREAD

**Issue by betterxu** [openyurtio/openyurt#1107]
**What happened**:
the other node is normal，when i use kubectl logs，only this node have error，i have rm tunnel-agent dir and restart，but，the error still exists.
Error from server: Get https://xxxxx:10250/containerLogs/kube-system/yurt-tunnel-agent-7xg8z/yurt-tunnel-agent?follow=true: dial tcp xxxx:10250: connect: connection timed out

> this is tunnel agent logs
`I1224 14:06:16.217175       1 start.go:49] yurttunnel-agent version: projectinfo.Info{GitVersion:"v0.7.1", GitCommit:"f6fb68c", BuildDate:"2022-08-30T14:38:44Z", GoVersion:"go1.17.1", Compiler:"gc", Platform:"linux/amd64"}
I1224 14:06:16.217651       1 options.go:148] ipv4=10.200.202.5&host=vm-202-5-centos is set for agent identifies
I1224 14:06:16.217663       1 options.go:153] neither --kube-config nor --apiserver-addr is set, will use /etc/kubernetes/kubelet.conf as the kubeconfig
I1224 14:06:16.217669       1 options.go:157] create the clientset based on the kubeconfig(/etc/kubernetes/kubelet.conf).
I1224 14:06:16.220583       1 start.go:86] yurttunnel-server address: xxxxxx:32000
W1224 14:06:16.220663       1 filestore_wrapper.go:49] unexpected error occurred when loading the certificate: no cert/key files read at "/var/lib/yurttunnel-agent/pki/yurttunnel-agent-current.pem", ("", "") or ("/var/lib/yurttunnel-agent/pki", "/var/lib/yurttunnel-agent/pki"), will regenerate it
I1224 14:06:16.220687       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Certificate rotation is enabled
I1224 14:06:16.220739       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Rotating certificates
I1224 14:06:17.225445       1 csr.go:188] error fetching v1 certificate signing request: the server could not find the requested resource
I1224 14:06:17.627273       1 csr.go:283] certificate signing request csr-9kkm5 is issued
I1224 14:06:18.827150       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Certificate expiration is 2023-12-24 14:01:16 +0000 UTC, rotation deadline is 2023-10-24 14:19:49.167659663 +0000 UTC
I1224 14:06:18.827220       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Waiting 7296h13m30.340444063s for next certificate rotation
I1224 14:06:19.827365       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Certificate expiration is 2023-12-24 14:01:16 +0000 UTC, rotation deadline is 2023-10-08 00:53:46.921425722 +0000 UTC
I1224 14:06:19.827391       1 certificate_manager.go:270] kubernetes.io/kube-apiserver-client: Waiting 6898h47m27.09403708s for next certificate rotation
I1224 14:06:21.221772       1 start.go:105] certificate yurttunnel-agent ok
I1224 14:06:21.221930       1 anpagent.go:57] start serving grpc request redirected from yurttunnel-server: 8.219.150.133:32000
I1224 14:06:21.222157       1 util.go:75] "start handling meta requests(metrics/pprof)" server endpoint="127.0.0.1:10266"
I1224 14:06:22.007848       1 client.go:224] "Connect to" server="ab5a1070-c5e8-47a0-aab1-3c9b06a98bf8"
I1224 14:06:22.007869       1 clientset.go:190] "sync added client connecting to proxy server" serverID="ab5a1070-c5e8-47a0-aab1-3c9b06a98bf8"
I1224 14:06:22.007895       1 client.go:326] "Start serving" serverID="ab5a1070-c5e8-47a0-aab1-3c9b06a98bf8"
I1224 14:06:27.910149       1 client.go:224] "Connect to" server="ac1ae4c8-f8fe-4ed6-bbec-43bf396a59f3"
I1224 14:06:27.910173       1 clientset.go:190] "sync added client connecting to proxy server" serverID="ac1ae4c8-f8fe-4ed6-bbec-43bf396a59f3"
I1224 14:06:27.910236       1 client.go:326] "Start serving" serverID="ac1ae4c8-f8fe-4ed6-bbec-43bf396a59f3"
I1224 14:06:34.009288       1 client.go:224] "Connect to" server="ac1ae4c8-f8fe-4ed6-bbec-43bf396a59f3"
E1224 14:06:34.009315       1 clientset.go:186] "closing connection failure when adding a client" err="client for proxy server ac1ae4c8-f8fe-4ed6-bbec-43bf396a59f3 already exists"
I1224 14:06:39.770591       1 client.go:224] "Connect to" server="cc260094-7b65-4f3c-8e96-69e9ea4ba4db"
I1224 14:06:39.770616       1 clientset.go:190] "sync added client connecting to proxy server" serverID="cc260094-7b65-4f3c-8e96-69e9ea4ba4db"
I1224 14:06:39.770644       1 client.go:326] "Start serving" serverID="cc260094-7b65-4f3c-8e96-69e9ea4ba4db"
`
**What you expected to happen**:
through tunnel，show pod logs
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- OpenYurt version: 0.7.1
- Kubernetes version (use `kubectl version`): 1.18.20
- OS (e.g: `cat /etc/os-release`): centos 7.9
- Kernel (e.g. `uname -a`): 3.10
- Install tools: kubeadm
- Others:

**others**
/kind question

**Comment by rambohe-ch** [comment]
@betterxu Sorry for late response because of covid-19. I want to check whether yurt-tunnel-server has two instances or not?

**Comment by betterxu** [comment]
> 

yes i have three instances,i check the original program,maybe it is a bug,beacuse,this node is my new node,the program shows if the node don‘t have tunnel agent pod and is ready will create a dnat iptables on master node,so the request will be  intercept。

**Comment by betterxu** [comment]
> response
yes i have three instances,i check the original program,maybe it is a bug,beacuse,this node is my new node,the program shows if the node don‘t have tunnel agent pod and is ready will create a dnat iptables on master node,so the request will be intercept。
